vms/platformvm/service.go

```diff
@@ -754,11 +754,13 @@ func (s *Service) GetCurrentValidators(_ *http.Request, args *GetCurrentValidato
 	reply.Validators = []interface{}{}
 
 	// Validator's node ID as string --> Delegators to them
-	vdrToDelegators := map[ids.NodeID][]platformapi.PrimaryDelegator{}
+	vdrToDelegators := map[ids.ShortNodeID][]platformapi.PrimaryDelegator{}
```

Comment:
```suggestion
	vdrToDelegators := map[ids.NodeID][]platformapi.PrimaryDelegator{}
```
I think this should be the general nodeID

Reply:
Done. Interestingly this requires NodeID to be a struct containing a string rather than a string alias. This is needed to let marshal machinery pick MarshalText to marshal NodeID.